dataplug-facebook/conf/silhouette.staging.conf

```diff
@@ -51,5 +51,5 @@ silhouette {
   facebook.clientSecret = ""
   facebook.redirectURL = ${?FACEBOOK_CLIENT_SECRET}
   facebook.clientSecret = ${?API_CLIENT_SECRET}
-  facebook.scope = "public_profile,user_friends,user_events,user_posts,email,user_age_range,user_link,user_gender,user_likes,user_birthday,user_location,user_photos,manage_pages"
+  facebook.scope = "public_profile,email,default,user_friends,user_gender,user_link,user_posts"
```

Comment:
Actually you can leave this as it was. The Dev app can have as many permissions as we want. It will also help us keep track to what we have tried fetching